web/cobrands/merton/base.scss

```diff
@@ -129,12 +163,52 @@ a.skiplink:focus {
     overflow: hidden;
     white-space: nowrap;
     padding-top: 60px;
-    width: 220px;
+    width: 150px;
+    background-position-y: 11px;
     height: 0;
     background: $merton-lavender-l2 url(/cobrands/merton/images/merton-logo.png) center center no-repeat;
     background-size: contain;
 }
 
+#main-nav {
+    overflow: visible;
+}
+
+.nav-menu {
+    background-color: $merton-jade-j3;
+    padding: 0.5em 0;
+
+    a, span {
+        background: transparent;
+        border-bottom: none;
+        font-size: 1em;
+    }
+
+    a:visited {
+        color: $merton-dark-green;
```

Comment:
This is causing a bug in the mobile nav when you hover over, you get dark green on grey, rather than white on grey as you get on unvisited links.

Reply:
Also this only changes it for visited, not all links as they asked in B16.